scripts/SDL2_gfx.sh

```diff
@@ -1,4 +1,8 @@
 test_deps_install SDL2
 get_pspport SDL_gfx SDL2_gfx-psp
+
+# This is used when there is no pkg-config, but needs sdl2-config
+#EXTRA_FLAGS="--with-sdl-prefix=$(psp-config --psp-prefix)"
```

Comment:
Should this be commented out? I see this in SDL2_image and SDL2_mixer as well.

Reply:
Yes, this is intentionally disabled until we ship `sdl2.pc` or `sdl2-config`.
When shipping the pkg-config file, we do not need to bend `SDL2_CFLAGS`/`LIBS` anymore, when only shipping the config script, this needs to be enabled.
`AM_PATH_SDL2` will use whatever available, with preference to pkg-config nowadays.

Reply:
Makes sense, thanks!